ComputerScienceHouse/vote2020
# Set Up Project for Service and Frontend Layers

Sets up the project to run api routes as well as the react frontend 

To test it out locally:  
- "npm start" should run the service on localhost:5000 and the react app on localhost:3000 
- when it's running, going to http://localhost:5000/api/getCount (or any of the other api routes) should show the response (They're just dummy responses for now) 
- going to localhost:5000 without an api route should serve the static files for the frontend (you'll need to build first, this also isn't super useful for locally bc I don't think sso will work, so this will be for deployed environments) 
- going to localhost:3000 should be the react app
